src/Hintman/Core/Hint.hs

```diff
@@ -10,6 +15,27 @@ module Hintman.Core.Hint
 import Relude.Extra.Enum (inverseMap)
 
 
+-- | Data type to represent the information for comment creation.
+data Hint = Hint
+    { hintHeader       :: !Text  -- ^ Hint text
+    , hintBody         :: !Text  -- ^ New line body
+    , hintIsSuggestion :: !Bool  -- ^ Is this GitHub @suggestion@?
```

Comment:
Am I right that we will get rid of this field when GitHub API supports multiline suggestions?

Reply:
Yes, this is correct :wink: